SpokestackTests/SharedTestMocks.swift

```diff
@@ -94,6 +94,8 @@ internal struct SharedTestMocks {
     ##38
     ##45
     million
+    [UNK]
+    Ålesund
```

Comment:
There shouldn't actually be a diacritic in the vocab file; what you're testing is lossless reconstruction of an input that wasn't in the BERT vocab.